config/locales/fr.yml

```diff
@@ -999,11 +1141,21 @@ fr:
       decimal_units:
         format: "%n%u"
         units:
-          billion: Milliard
-          million: Million
-          quadrillion: Quadrillion
-          thousand: Mille
-          trillion: Trillion
+          billion:
```

Comment:
I checked the Transifex project and these words are not translated there, so the PR kind of overwrites the existing translations. I guess this is because these words were directly added to the yml file and not translated via transifex. An easy fix is to add these into transifex and then do the sync once more.

Reply:
Ah, hmm. So we would have to anticipate this happening again and add an optional prior step of copying in any manually-introduced strings into Transifex?